noxfile.py

```diff
@@ -72,6 +72,7 @@ def default(session):
     )
 
 
+@nox.session(python=["3.6", "3.7", "3.8", "3.9", "3.10"])
```

Comment:
This package has no tests, so it instead runs the unit and system tests of client libraries that take a dependency instead.

This session not meant to be directly run as a session. If you look below at `test`, it calls `unit()` and `system()`.